src/components/pages/blog-post/blog-post.tsx

```diff
@@ -95,18 +97,12 @@ const contentfulRenderOptions: Options = {
         case 'image': {
           return customComponents.figure({
             children: [
-              customComponents.a({
-                key: 'link',
-                children: (
-                  <GatsbyImage
-                    image={node.data.target.gatsbyImageData}
-                    alt={node.data.target.description}
-                  />
-                ),
-                href: node.data.target.file.url,
-                target: '_blank',
-                rel: 'nofollow noopener noreferrer',
-              }),
+              <Zoom key={node.data.target.file.url}>
```

Comment:
I think that should do it for blog posts 👍
 @dschu-lab as you have worked on that mapping recently, what's your opinion on that? Good to add the zoom image entry here for every image in the blog post by default?

Reply:
I like it! Let's get this merged to main. 🙌